src/main/arm/createUiDefinition.json

```diff
@@ -93,17 +86,17 @@
                         "type": "Microsoft.Common.TextBox",
                         "label": "Number of VMs",
                         "defaultValue": "4",
-                        "toolTip": "The number of VMs to create, with one deployment manager and multiple worker nodes for the remainings.",
+                        "toolTip": "The number of virtual machines to create, with 1 deployment manager and 1 to 4 worker nodes.",
                         "constraints": {
                             "required": true,
                             "regex": "^(2|3|4|5)$",
-                            "validationMessage": "Number of VMs to deploy, limit 5."
+                            "validationMessage": "Number of virtual machines to deploy, limit 5 since this offer is using a single storage account."
```

Comment:
Please change "limit 5 since this offer is using a single storage account" to "limit 5 because this offer uses a single storage account"

Reply:
@helyarp Done in commit 7cae876.

BTW, I also added back tooltips in this commit as they're required by Azure template unit tests.